src/main/java/no/nav/veilarboppfolging/repository/BrukerOppslagFlereOppfolgingAktorRepository.java

```diff
@@ -0,0 +1,43 @@
+package no.nav.veilarboppfolging.repository;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.veilarboppfolging.utils.DbUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Slf4j
+@Repository
+public class BrukerOppslagFlereOppfolgingAktorRepository {
+    private final JdbcTemplate db;
+
+    @Autowired
+    public BrukerOppslagFlereOppfolgingAktorRepository(JdbcTemplate db){
+        this.db = db;
+    }
+
+    public void insertBrukerOppslag(String norskIdent){
+        long id = DbUtils.nesteFraSekvens(db, "BRUKER_OPPSLAG_MED_FLERE_AKTORID_SEQ");
+        if (getBrukerLogget(norskIdent)) {
+            return;
+        }
+
+        db.update("INSERT INTO BRUKER_OPPSLAG_MED_FLERE_AKTORID(" +
+                        "BRUKER_SEQ, " +
+                        "OPPSLAG_BRUKER_ID, " +
+                        "CREATED) " +
+                        "VALUES(?, ?, CURRENT_TIMESTAMP)",
+                id,
+                norskIdent);
+    }
```

Comment:
her risikerer du at to prøver ogs sette inn samtidig. 
telleren vil også gå opp selv om det ikke blir satt inn en  bruker

Reply:
er det smartere og bare sette inn og catche riktig exception?

Reply:
Jeg foretrekker å gjøre sjekk isteden for exception håndtering, iom at jeg vet at det er en vanlig situasjon at brukeren allerede ligger der. Godt poeng med sekvensgreiene, skal endre rekkefølgen. Hva mener du med at flere kan sette inn samtidig? Mener du samme brukeren i forskjellige podder?

Reply:
Hvis jeg putter en transactional på denne så løser jeg problemet?